modules/configuring-clusterqueues.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * /configure/configuring-quotas.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-clusterqueues_{context}"]
+= Configuring a cluster queue
+
+A cluster queue is a cluster-scoped resource, represented by a `ClusterQueue` object, that governs a pool of resources such as CPU, memory, and pods.
```

Comment:
In the "configuring-quotas" you mention GPUs but then your example focuses on CPU, memory and pods.